package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think the package.json and package-lock.json at the root directory should be deleted.